renovate.json

```diff
@@ -2,5 +2,8 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "github>cloudeteer/terraform-governance:renovate-default.json"
-  ]
+  ],
+  "pre-commit": {
+    "enabled": true
+  }
```

Comment:
Enable pre-commit in `governance/renovate-default.json`

Reply:
Not sure, if that works with cloud renovate

Reply:
Thanks for the hint, @jkroepke. I will investigate this.

Reply:
Will be handled in #5

Reply:
Will be covered in https://github.com/cloudeteer/terraform-governance/issues/24